packages/picker/src/Picker.ts

```diff
@@ -60,6 +60,18 @@ const chevronClass = {
 export class PickerBase extends SizedMixin(Focusable, { noDefaultSize: true }) {
     protected isMobile = new MatchMediaController(this, IS_MOBILE);
 
+    static instanceCount = 0;
+    instanceCount: number;
+
+    @property({ type: String })
+    public description = '';
+
+    constructor() {
+        super();
+        this.instanceCount = PickerBase.instanceCount++;
+        this.description = `sp-picker-${this.instanceCount}`;
```

Comment:
This description is being completed within the shadow root, we should be able to use a static value here, rather than constructing something with the instance count.

Reply:
done!